js/ui.js

```diff
@@ -1,16 +1,169 @@
-$(document).ready(function() {
-  $("#tabs").tabs();
-});
 
-$('#user-menu-trigger').on('click', function() {
-  if ($('.dropdown').is(':visible')) {
-    $('.dropdown').slideUp();
-  } else {
-    $('.dropdown').slideDown();
-    $('#first-dropdown-link').focus();
-  }
+
+$(document).ready(function(){
+	$('#tabs').tabs({'active':1});
+	$('#tabs' ).tabs('disable', 0 );
+	$('#tabs' ).tabs('disable', 3 );
+    $('.alert-container').hide();
```

Comment:
Sorry, it was just a quick fix for alert-container blocking Leaflet events. I'll fix it.